apps/css-workshop/pages/tag.html

```diff
@@ -733,38 +733,38 @@ <h2>Basic tag containers</h2>
       <div class="iui-tag-container">
         <a
           href="#"
-          class="iui-tag-basic"
+          class="iui-anchor iui-tag-basic"
```

Comment:
nit: Looks like there's some extra leading and trailing space.

![image](https://github.com/iTwin/iTwinUI/assets/45748283/f5ca0e12-2daf-42f7-bedc-ccbea6a649da)

![image](https://github.com/iTwin/iTwinUI/assets/45748283/44fbc4e6-1393-4a21-9cb9-9b56d614a2ae)

I'm wondering how these weren't pointed out by the visual tests?

Reply:
Actually, this can be be ignored since it's likely an issue on my side. Because [this PR's deployed version's `tag.html`](https://itwin.github.io/iTwinUI/1915/css/tag.html) shows up correctly.